udata/settings.py

```diff
@@ -462,6 +462,8 @@ class Defaults(object):
                         'IssueAPI.post',
                         'SourcesAPI.post',
                         'FollowAPI.post']
+    
+    DATASET_SLUGS = []
```

Comment:
Could we name it more explicitely: `FIXTURE_DATASET_SLUGS` maybe?

And could you add a small entry in `adapting-settings.md` ?